src/components/Link.tsx

```diff
@@ -1,5 +1,10 @@
 import { Link as ChakraLink, LinkProps } from '@chakra-ui/react'
+import { forwardRef } from 'react'
 
-const Link = (props: LinkProps) => <ChakraLink color="primary.500" {...props} />
+const Link = forwardRef<HTMLAnchorElement, LinkProps>((props, ref) => (
```

Comment:
I'm still having problems using <kbd>tab</kbd> with Firefox `94.0`.
I can now only focus on the big "Go to comparator" button with the keyboard, not on the "Or go to homepage" text, or on the h1 navbar, or even the links in the footer.

So something is still broken here. 😕

Reply:
I don't see more issues navigating with the keyboard. This is on firefox 94:

https://user-images.githubusercontent.com/2677072/140300984-68173712-23fa-4081-baa4-1d564d35ade0.mp4

This might be a problem on your end. Make sure you have enabled the browser keyboard navigation if you are using macOS: https://www.a11yproject.com/posts/2017-12-29-macos-browser-keyboard-navigation/

I already had this + always visible scrollbar in order to test websites properly. I just checked that disabling that option I can't navigate to the links either, so it must be definitely that.

Reply:
> This might be a problem on your end. Make sure you have enabled the browser keyboard navigation if you are using macOS: https://www.a11yproject.com/posts/2017-12-29-macos-browser-keyboard-navigation/

Problem was on my end. 🙈 Fixed now by following the instructions in the link! 🥳

Reply:
This has been a huge TIL for me by the way! I didn't know that option could affect the keyboard navigation in websites, I thought it was meant to be only for macOS dialogs (which is the reason I enabled the setting).